build/package/.mongocli.goreleaser.yml

```diff
@@ -86,8 +84,23 @@ changelog:
     - '^test:'
     - '^task:'
     - '^build(deps):'
+publishers:
```

Comment:
[q] doesn't this run in snapshots?

Reply:
It does not

```bash
  • skipped publishing
  • storing release metadata
    • writing                                        file=dist/artifacts.json
    • writing                                        file=dist/metadata.json
  • skipped announcing
  • you are using deprecated options, check the output above for details
  • release succeeded after 15s
```

Reply:
today we sign snapshot binaries, this is important if we find ourselves in a position to give binaries to someone for testing (we've done this before)

Reply:
discussed this offline, we will prelease the provider to cover this scenario similar to https://github.com/andreaangiolillo/mongocli-test/releases/tag/v1.22.43-pre